docs/conf.py

```diff
@@ -77,7 +82,7 @@ def fixpath(p):
     get away with having only one copy of the CSS and web fonts.
     """
     new = f"/{project}/latest/_static/" + p.split("_static")[1]
-    return new
+    return p
```

Comment:
Is the variable `new` still needed then? What is the nature of this change?

Reply:
This section needs work. This was a hack to get it working for, but we should figure out exactly what's going on here and why it doesn't work before merging